contentctl/objects/enums.py

```diff
@@ -405,14 +407,16 @@ class NistCategory(str, enum.Enum):
     RC_IM = "RC.IM"
     RC_CO = "RC.CO"
 
-class RiskLevel(str,enum.Enum):
-    INFO = "Info"
-    LOW = "Low"
-    MEDIUM = "Medium"
-    HIGH = "High"
-    CRITICAL = "Critical"
-
 class RiskSeverity(str,enum.Enum):
+    # Levels taken from the following documentation link
+    # https://docs.splunk.com/Documentation/ES/7.3.2/User/RiskScoring
+    # 20 - info (0-20 for us)
+    # 40 - low (21-40 for us)
+    # 60 - medium (41-60 for us)
+    # 80 - high (61-80 for us)
+    # 100 - critical (81 - 100 for us)
+    INFO = "info"
```

Comment:
This value was previously a hardcoded `high` in all our detections.
As such, I have made them all lowercase - does the capitalization matter?

Reply:
We should run a couple integration tests to confirm that this works- need to do the always_pause deal and examine Incident Review in the ES app to confirm.

Reply:
Running a test, it looks like the label `info` does not work. Low, Medium, High, and Critical both render in the UI:
<img width="1146" alt="image" src="https://github.com/user-attachments/assets/861dcade-6d07-48b1-855e-195de888fa7a">

I am guessing that instead of info, it need to be Informational based on this screenshot:
<img width="1146" alt="image" src="https://github.com/user-attachments/assets/15f108bb-bd01-42ca-b0c9-5e99b2131f94">

Reply:
With this update: https://github.com/splunk/contentctl/pull/179/commits/b488c22438a86f47b5ec55441b598843c3d6e1d3#diff-3092f62aa5aec6268a6d7e75992b8830e170eae9bbc6b4e4ec20413ebc6ecae0R418


It looks like all Severity Levels are now generated Properly (`Unknown` is not supported by contentctl, so it is not shown as an example):
<img width="2929" alt="image" src="https://github.com/user-attachments/assets/6794379d-c33a-43e2-9ee0-6d4095bc5de0">